.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "Csocket"]
+	path = third_party/Csocket
+	url = https://github.com/jimloco/Csocket
```

Comment:
I am not familiar with submodules, but shouldn't this be `https://github.com/jimloco/Csocket.git`? GitHub works even without the .git, but it might be confusing to people who are used to .git always being there.